helix-lsp/Cargo.toml

```
@@ -19,14 +19,14 @@ helix-loader = { path = "../helix-loader" }
 helix-lsp-types = { path = "../helix-lsp-types" }
 
 anyhow = "1.0"
-futures-executor = "0.3"
-futures-util = { version = "0.3", features = ["std", "async-await"], default-features = false }
+futures-executor.workspace = true
+futures-util.workspace = true
 globset = "0.4.16"
 log = "0.4"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 tokio = { version = "1.45", features = ["rt", "rt-multi-thread", "io-util", "io-std", "time", "process", "macros", "fs", "parking_lot", "sync"] }
-tokio-stream = "0.1.17"
+tokio-stream.workspace = true  
```

```suggestion
tokio-stream.workspace = true
```

looks like some trailing whitespace snuck in here